geoips/commandline/log_setup.py

```diff
@@ -33,10 +33,17 @@ def log_with_emphasis(print_func, *messages):
         The messages to be logged with emphasis
     """
     wrapped_messages = []
+    messages = filter(lambda s: len(s) > 0, messages)
```

Comment:
Thanks! @evrose54 ^ because you brought it up I realized that I didn't need to cast to a list here :)

Reply:
No worries! Glad this is working as expected. Going to resolve this conversation now.